tools/provision/chocolatey/aws-sdk-cpp.ps1

```diff
@@ -6,16 +6,16 @@
 #  of patent rights can be found in the PATENTS file in the same directory.
 
 # Update-able metadata
-$version = '1.0.107'
-$chocoVersion = '1.0.107-r1'
+$version = '1.1.44'
+$chocoVersion = '1.1.44-r1'
```

Comment:
If this is the first time we've seen this version, the `-r1` isn't required and we can do whole version releases, mind removing this?